core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -1,25 +1,4 @@
 <script type="text/ng-template" id="skins/Conversation">
-  <!--
```

Comment:
Note that this "spare" tutor card used to be around in order to pre-determine the height change. Is it not needed any more?